uplc-discharge.md

```diff
@@ -43,7 +47,13 @@ module UPLC-DISCHARGE
 
   rule dischargeTerm((builtin BN:BuiltinName), _) => (builtin BN)
 
-  rule dischargeTerm((lam X:UplcId T:Term), RHO:Map) => (lam X dischargeTerm(T, RHO))
+  rule dischargeTerm((lam X:UplcId T:Term), RHO:Map) =>
+       (lam X dischargeTerm(T, RHO[X <- undef]))
+  requires X in_keys(RHO)
```

Comment:
Same comment as above

Reply:
Yes, map delete is total. I will change it.